lib/engines/hcktest/hcktest.rb

```diff
@@ -274,7 +281,7 @@ def run_tests_with_config
         next if tests.empty?
 
         ResourceScope.open do |scope|
-          run_clients_and_configure_setup(scope, group => true, create_snapshot: false, boot_from_snapshot: true)
+          run_clients_and_configure_setup(scope, create_snapshot: false, group => true)
```

Comment:
Why did you change this?

Reply:
My question is not about session restoration, but about the normal case that does not involve session restoration. This change will affect all scenarios since it does not check `project.restored?`; is that OK?